src/man/firejail.txt

```diff
@@ -2142,6 +2143,7 @@ track of CPU seconds for each process independently.
 .TP
 \fB\-\-rlimit-fsize=number
 Set the maximum file size that can be created by a process.
+you can use kilobyte(k),megabyte(m) and gigabyte(g) for size suffix. ( they works on base 1024 )
```

Comment:
suggestion:
 Use k(ilobyte), m(egabyte) or g(igabyte) for size suffix (base 1024).